docs/json_schemas/dns_record/v0/requirer.json

```diff
@@ -0,0 +1,227 @@
+          },
+          "title": "Dns Domains",
+          "type": "array"
+        },
+        "dns_entries": {
+          "default": null,
+          "description": "List of DNS records for the provider to manage.",
+          "items": {
+            "$ref": "#/$defs/RequirerEntries"
+          },
+          "title": "Dns Entries",
+          "type": "array"
+        }
+      },
+      "title": "DNSRecordRequirer",
+      "type": "object"
+    },
+    "RecordClass": {
+      "const": "IN",
+      "description": "Represent the DNS record classes.",
+      "title": "RecordClass",
+      "type": "string"
+    },
+    "RecordType": {
+      "description": "Represent the DNS record types.",
+      "enum": [
+        "A",
+        "AAAA",
+        "CNAME",
+        "MX",
+        "DKIM",
+        "SPF",
+        "DMARC",
+        "TXT",
+        "CAA",
+        "SRV",
+        "SVCB",
+        "HTTPS",
+        "PTR",
+        "SOA",
+        "NS",
+        "DS",
+        "DNSKEY"
+      ],
+      "title": "RecordType",
+      "type": "string"
+    },
+    "RequirerDomains": {
+      "properties": {
+        "domain": {
+          "description": "Domain name for the provider to manage.",
+          "examples": [
+            "cloud.canonical.com",
+            "staging.ubuntu.com"
+          ],
+          "minLength": 1,
+          "name": "Domain",
+          "title": "Domain",
+          "type": "string"
+        },
+        "username": {
+          "description": "Username for authentication.",
+          "examples": [
+            "user1",
+            "user2"
+          ],
+          "name": "Username",
+          "title": "Username",
+          "type": "string"
+        },
+        "password": {
+          "description": "Juju secret containing the user password.",
+          "examples": [
+            "secret:123213123123123123123"
+          ],
+          "name": "Password",
+          "title": "Password",
+          "type": "string"
+        },
+        "uuid": {
+          "description": "UUID for this domain.",
+          "examples": "550e8400-e29b-41d4-a716-446655440000",
+          "minLength": 1,
+          "name": "UUID",
+          "title": "Uuid",
+          "type": "string"
+        }
+      },
+      "required": [
+        "domain",
+        "username",
+        "password",
+        "uuid"
+      ],
+      "title": "RequirerDomains",
+      "type": "object"
+    },
+    "RequirerEntries": {
+      "properties": {
+        "domain": {
+          "description": "Domain name requested.",
+          "examples": [
+            "cloud.canonical.com",
+            "staging.ubuntu.com"
+          ],
+          "minLength": 1,
+          "name": "Domain",
+          "title": "Domain",
+          "type": "string"
+        },
+        "host_label": {
+          "examples": [
+            "admin",
+            "www"
+          ],
+          "minLength": 1,
+          "name": "Host label",
+          "status_description": "Host label.",
```

Comment:
why do we have `status_description` keys all over the place, shouldn't those be `description`?

Reply:
Probably a replace all but indeed should be description